CodeGeneration/Sources/SyntaxSupport/Child.swift

```diff
@@ -25,16 +25,19 @@ public enum TokenChoice: Equatable {
     }
   }
 
-  public var varOrCaseName: TokenSyntax {
+  /// The name of this token choice as an identifier.
+  public var identifier: TokenSyntax {
     switch self {
     case .keyword(let keyword):
-      return keyword.spec.varOrCaseName
+      return keyword.spec.identifier
     case .token(let token):
-      return token.spec.varOrCaseName
+      return token.spec.identifier
     }
   }
 }
 
+extension TokenChoice: IdentifierConvertible {}
```

Comment:
Could we add these conformances to the type definition itself? I think that makes it easier to discover which protocols a type conforms to, if the conformance is not an extension. Same for the other conformances.

Reply:
Sure, lemme do it.